include/roboteam_utils/Angle.h

```diff
@@ -12,218 +12,117 @@ namespace rtt {
-         */
-        Angle operator-(const double &scalar) const noexcept;
-
-        /**
-         * @brief Sets the current angle equal to`this->angle + other.angle`
-         *
-         * @param other Other angle to add to `this`
-         * @return Angle A copy of `*this`
+         * Add the other Angle to this Angle, and directly rescale the angle value.
+         * @param other The other Angle.
+         * @return A reference to this updated Angle instance.
          */
         Angle operator+=(const Angle &other) noexcept;
 
         /**
-         * @brief Sets the current angle equal to `this->angle + scalar`
-         *
-         * @param scalar Other angle to add to `this`
-         * @return Angle A copy of `*this`
-         */
-        Angle operator+=(const double &scalar) noexcept;
-
-        /**
-         * @brief Sets the current angle equal to `this->angle - other.angle`
-         *
-         * @param other Other angle to subtract from `this`
-         * @return Angle A copy of `*this`
+         * Subtract the other Angle from this Angle, and directly rescale the angle value.
+         * @param other The other Angle.
+         * @return A reference to this updated Angle instance.
          */
         Angle operator-=(const Angle &other) noexcept;
 
         /**
-         * @brief Sets the current angle equal to `this->angle - scalar`
-         *
-         * @param scalar Other angle to subtract from `this`
-         * @return Angle A copy of `*this`
-         */
-        Angle operator-=(const double &scalar) noexcept;
-
-        /**
-         * @brief Copy assignment operator
-         *
-         * Explicitly constraints.
-         *
-         * @param scalar Scalar to set `this->angle` to
+         * Set the angle value of this Angle instance, which will be directly rescaled.
+         * @param scalar A double value which represent the new angle value. It is allowed to use a value outside the range [-PI, PI). However you should be cautious with float
+         * values close to -MAX_FLOAT and close to MAX_FLOAT.
+         * @return A reference to this updated Angle instance.
          */
         Angle &operator=(const double &scalar) noexcept;
 
-        /**
-         * @brief Convert the Angle to a double.
-         *
-         * TODO: explicit
-         *
-         * @return double `this->angle`
-         */
-        operator double() const noexcept;
-
     private:
-        /**
-         * @brief Current angle stored
-         */
-        double angle;
+        double angle; // The current angle value, which is a value in the range [-PI, PI).
 
         /**
-         * @brief Constrains the angle between 0 and 2 pi
-         *
-         * @return Angle A copy of `*this`
+         * Rescale the angle value to the range [-PI, PI).
+         * @return A reference to this rescaled Angle instance.
          */
-        Angle constrain() noexcept;
+        Angle rescale() noexcept;
```

Comment:
I personally like the old name more. Rescaling sounds like you´re doing a transformation on the angle (e.g. angles to degrees or multiplying it by some number), which you are not . I like the names 'normalize' or 'constrain' better as they are clearer (atleast, to me) and a bit more clear in what the function is actually doing.

Reply:
Well normalize sound better to me than constrain